<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TBSetupPayoutsWidget</class>
 <widget class="QWidget" name="TBSetupPayoutsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Payouts</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="payoutTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="manualTab">
      <attribute name="title">
       <string>Manual</string>
      </attribute>
      <layout class="QVBoxLayout" name="manualLayout">
       <item>
        <widget class="QTableView" name="manualTableView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="manualButtonLayout">
         <item>
          <widget class="QPushButton" name="addPayoutButton">
           <property name="text">
            <string>Add Payout</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removePayoutButton">
           <property name="text">
            <string>Remove Payout</string>
           </property>
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="manualButtonSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="turnoutTab">
      <attribute name="title">
       <string>Depends on Turnout</string>
      </attribute>
      <layout class="QVBoxLayout" name="turnoutLayout">
       <item>
        <widget class="QSplitter" name="turnoutSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableView" name="turnoutTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTableView" name="turnoutPayoutsTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="turnoutButtonLayout">
         <item>
          <widget class="QPushButton" name="addTurnoutButton">
           <property name="text">
            <string>Add Turnout Level</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeTurnoutButton">
           <property name="text">
            <string>Remove Level</string>
           </property>
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="turnoutButtonSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>